modules/ballerina-builtin/pom.xml

```diff
@@ -58,6 +58,11 @@
             <groupId>org.apache.ws.commons.axiom</groupId>
             <artifactId>axiom-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.ecf</groupId>
```

Comment:
Since this is adding a new dependency to a ballerina, you have to send a PR to tools distribution by packing this dependency.